### PR TITLE
add support for flowdock

### PR DESCRIPTION
**Pull request by dgolja**
Similar as Slack I added support for Flowdock

### CODE REVIEW

File: flowdock.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+)
+
+type flowDockNotifier struct {
+	Flow_token string
```
Comment (dhowden) [inline flowdock.go]: Shouldn't have underscores in variable names (follow Go naming conventions).

**Comment by dhowden** [comment]
Awesome!  Thanks.  If you fix that variable name, I'll merge it straight in!

**Comment by dgolja** [comment]
Yes sensei :)

**Comment by dhowden** [comment]
Awesome, thanks!

**Pull request merged**